src/main/haskell/language-kore/src/Data/Kore/ASTPrettyPrint.hs

```diff
@@ -118,25 +126,28 @@ writeStructure name fields =
 printableList :: PrinterOutput w m => [m ()] -> [m ()]
 printableList = intersperse (betweenLines >> write ", ")
 
-instance (MetaOrObject level) => PrettyPrint (Id level) where
-    prettyPrint flags id'@(Id name) =
+instance MetaOrObject level => PrettyPrint (Id level) where
+    prettyPrint flags x =
         betweenParentheses
             flags
             (do
                 write "Id "
-                write
-                    (applyMetaObjectFunction
-                        id'
-                        MetaOrObjectTransformer
-                            { metaTransformer = const "Meta"
-                            , objectTransformer = const "Object"
-                            }
-                    )
-                write " \""
-                write name
                 write "\""
+                write (getId x)
```

Comment:
Thanks for fixing this. At some point we should use the `flags` to display this only when it can't be inferred from an including construct.

Reply:
Agreed.  I'll add a TODO